packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -14,19 +14,24 @@ import 'timeline_protocol.dart';
 
 class FramesBarChart extends CoreElement {
   FramesBarChart(TimelineController timelineController)
-      : super('div', classes: 'timeline-frames section-border') {
+      : super('div', classes: 'timeline-frames') {
```

Comment:
maybe add a comment indicating why we don't want a border around the bar chart area unlike most other ui components we have?

Reply:
Done